packages/azure-http-specs/specs/azure/client-generator-core/client-initialization/main.tsp

```diff
@@ -0,0 +1,120 @@
+import "@typespec/http";
+import "@typespec/spector";
+import "@azure-tools/typespec-client-generator-core";
+
+using Http;
+using Spector;
+using Azure.ClientGenerator.Core;
+
+@doc("Test for client initialization decorator - moving parameters from method to client level")
+@scenarioService("/azure/client-generator-core/client-initialization")
+namespace Service;
+
+// Common parameter types and models
+
+model NameHeaderParam {
+  @header("name")
+  name: string;
```

Comment:
Curious why there is no change in spec-summary.md after new cases added.
https://github.com/Azure/typespec-azure/blob/c2d118aa8ac206a25537839fe8cf0fa9f422c40f/packages/azure-http-specs/spec-summary.md

Reply:
i think doc is relay on `@scenario`. and izzy put it on interface level, not operation level.